### PR TITLE
Update installation and upgrade steps

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by arihantg** [comment]
skip jira-id-check

File: README.md [hunk 1]
```diff
@@ -159,15 +159,15 @@ kubectl apply -f <output_dir>/amko/crds/
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED                             	    STATUS  	CHART    	APP VERSION
-amko-1598451370 avi-system	1       	2022-05-19 10:00:31.609195757 +0000 UTC	    deployed	amko-1.9.1	1.9.1
+amko-1598451370 avi-system	1       	2022-05-19 10:00:31.609195757 +0000 UTC	    deployed	amko-1.10.1	1.10.1
```
Comment (pkoshtavmware) [inline README.md:162]: Might want to change updated date here as well as.
As per this document https://confluence.eng.vmware.com/pages/viewpage.action?pageId=1624722802, AKO/AMKO 1.10.1 got GAed on 9 June 2023

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: README.md [hunk 2]
```diff
@@ -97,14 +97,14 @@ Following steps have to be executed on all member clusters:
...
-   $ helm install --generate-name oci://projects.registry.vmware.com/ako/helm-charts/amko --version 1.10.1 -f /path/to/values.yaml  --set configs.gsllbLeaderController=<leader_controller_ip> --namespace=avi-system
+   $ helm install --generate-name oci://projects.registry.vmware.com/ako/helm-charts/amko --version 1.11.1 -f /path/to/values.yaml  --set configs.gsllbLeaderController=<leader_controller_ip> --namespace=avi-system
    ```
 6. Check the installation:
    ```
    $ helm list -n avi-system
 
    NAME           	NAMESPACE 	REVISION	UPDATED                                	STATUS  	CHART                 	APP VERSION
-   amko-1598451370	avi-system	1       	2022-02-04 11:16:21.889538175 +0000 UTC	deployed	amko-1.10.1	            1.10.1
+   amko-1598451370	avi-system	1       	2022-02-04 11:16:21.889538175 +0000 UTC	deployed	amko-1.11.1	            1.11.1
```
[the remark this answers is not in the document]
Comment (arihantg) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (arihantg) [reply README.md]: updated

**Comment by arihantg** [comment]
skip jira-id-check

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Pull request merged**